changelog.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+### Added
```

Comment:
This probably comes under 'removed', I think.

Reply:
I'll address this on #10